modules/gitops-release-notes-1-13-0.adoc

```diff
@@ -0,0 +1,119 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-13-0_{context}"]
+= Release notes for {gitops-title} 1.13.0
+
+{gitops-title} 1.13.0 is now available on {OCP} 4.12, 4.13, 4.14, 4.15, and 4.16.
+
+[id="errata-updates-1-13.0_{context}"]
+== Errata updates
+
+[id="RHSA-XXXX:NNNN-gitops-1-13-0-security-update-advisory_{context}"]
+=== RHSA-XXXX:NNNN - {gitops-title} 1.13.0 security update advisory
+
+Issued:
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-XXXX:NNNN[RHSA-XXXX:NNNN]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="new-features-1-13-0_{context}"]
+== New features
+
+* With this update, you can configure Argo CD to disable the automatic scraping of metrics for your instance. Before this update, there was no option to disable this feature, resulting in excessive storage usage when multiple Argo CD instances were available on the cluster. link:https://issues.redhat.com/browse/GITOPS-4519[GITOPS-4519]
++
+[NOTE]
+====
+In {gitops-shortname}, the scraping of metrics is enabled by default for all Argo CD instances.
+====
++
+To disable metrics for your Argo CD instance, edit the `spec.monitoring.disableMetrics` field in the Argo CD custom resource to `false`.
```

Comment:
Update the last part to - 

field in the ArgoCD CR to `true`
instead of
Argo CD custom resource to `false`

Reply:
Fixed.